src/storage/manager.rs

```diff
@@ -17,11 +17,21 @@ use crate::{
 /// The storage used by the manager.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub(crate) enum ManagerStorage {
-    #[cfg(feature = "stronghold")]
     /// Stronghold storage.
+    #[cfg(feature = "stronghold")]
     Stronghold,
     /// RocksDB storage.
+    #[cfg(feature = "rocksdb")]
     Rocksdb,
+    /// Storage backed by a Map in memory.
+    Memory,
+}
+
+pub(crate) const fn default_storage() -> ManagerStorage {
+    #[cfg(feature = "rocksdb")]
+    return ManagerStorage::Rocksdb;
+    #[cfg(not(feature = "rocksdb"))]
+    ManagerStorage::Memory
```

Comment:
Can this not be an impl Default ?

Reply:
Not for an enum no, unfortunately 😢

Reply:
What do you mean ? What's the issue with 
```rust
impl Default for ManagerStorage {
    fn default() -> ManagerStorage {
        #[cfg(feature = "rocksdb")]
        return ManagerStorage::Rocksdb;
        #[cfg(not(feature = "rocksdb"))]
        ManagerStorage::Memory
    }
}
```

Reply:
oh.. why did that not come up in my brain... i was thinking about derive default and cant do that in enums haha

Reply:
You actually can https://rust-lang.github.io/rfcs/3107-derive-default-enum.html, I'm just not sure of the syntax together with feature gates but it should be doable. But fine to impl anyway.